eth-types/src/sign_types.rs

```diff
@@ -139,6 +231,18 @@ impl Default for SignData {
     }
 }
 
+// Default for secp256r1
+impl Default for SignData<Fq_R1, Secp256r1Affine> {
+    // Hardcoded valid signature corresponding to a hardcoded private key and
+    // message hash generated from "nothing up my sleeve" values to make the
```

Comment:
message hash generated from "nothing up my sleeve": is this valid?  where is the hashing of this example data ""nothing up my sleeve""?

Reply:
it is for padding sig circuit rows, the message hash is not from real world tx but valid for signature verification. in other words. just found  pair value ( msg_hash, private key etc)  and get a valid signature.      for k1 default padding data, refer to https://github.com/scroll-tech/zkevm-circuits/blob/feat/secp256r1/eth-types/src/sign_types.rs#L223, which use dummy tx with a specified private key and sign to get a valid signature.